contributing/code/standards.rst

```diff
@@ -52,12 +54,18 @@ short example containing most features described below::
          */
         private $fooBar;
 
+        /**
+         * @var Qux
```

Comment:
actually, when the constructor already hints about the type, we recommend not duplicating in a code block.